sections/Discussion.tex

```diff
@@ -0,0 +1,87 @@
+		}
+	\end{tcolorbox}
+	\caption{A random article drawn from our dataset.}
+	\label{fig:the_article}
+\end{figure}
+
+Overall, we see that there is a large amount of overlap between the models, which is interesting since the models use different meta-information to create the various topic distributions.
+This indicates that the models share many of the top words, while also indicating a slight deviation between the models due to the meta-information.
+The \gls{lda} model shows words like "landbrug" (agriculture) and "produktion" (production), which is what the article is mostly about.
+This behavior is to be expected since the performance of \gls{lda} has been explored and evaluated before. 
+Author-topic specific words are not very present and are only showing three unique words: "byder", "hører", and "børnene".
+This indicates that the Author-topic model has trouble generalizing what the author of this article (Peter Tordrup Larsen) is writing about, possibly because he has written $5002$ articles in our dataset.
+Another aspect of the Author-Topic model is that the authors writing these articles most likely do not write about just one subject, which explains why there is only three less important words marked here. 
+The Category-topic model only shows four unique words: "søndag" (sunday), "leg" (play), "klokken" (clock), and "danskere" (Danes).
+These words are also very abstract and can be used in many different scenarios.
+
+An interesting part of this analysis is the words appearing in all models.
+Some of the words within this category are: "dansk" (Danish), "millioner" (millions), "samt" (including), and "nordjyske" (North Jutland).
+These words are not very representative of the content of the article, but it is also hard to summarize this text since it covers a wide variety of specific topics.
```

Comment:
```suggestion
These words are not very representative of the content of the article, but the article is also difficult to summarize since it covers a wide variety of specific topics.

```